docs/reference/swagger-self-hosted.md

```diff
@@ -0,0 +1,36 @@
+# Swagger (self-hosted)
+
+Wharf's main API and provider APIs all produce OpenAPI specifications.
```

Comment:
shouldn't be here provider**s** APIs?

Reply:
I've no idea. I'm used to only pluralize the last word. If that's correct or not I have no clue

Reply:
K if I resolve?

Reply:
I am ok with it.